UI/data/themes/Yami.obt

```diff
@@ -1413,7 +1413,7 @@ VolControl #volLabel {
     margin-right: var(--padding_xlarge);
 }
 
-#vMixerScrollArea VolControl MuteCheckBox {
+#vMixerScrollArea VolControl indicator-mute {
```

Comment:
I this supposed to not have a preceding `.`?

Reply:
Yes. Interesting that this didn’t show up in testing, maybe it was also broken before in some way.
Will fix tomorrow.